examples/resources/dsm_aws_sobject/resource.tf

```diff
@@ -1,53 +1,55 @@
+/*
+How to create an AWS KMS key with static credentials ?
```

Comment:
Since this is an example we can omit the `?` at the end

Reply:
Done.